hw/ip/padctrl/data/padctrl_fpv_testplan.hjson

```diff
@@ -0,0 +1,167 @@
+{
+  name: "padctrl"
+  import_testplans: ["hw/dv/tools/testplans/fpv_csr_testplan.hjson"]
+  entries: [
+    {
+      name: MioWarl_A
+      desc: '''In either generic or Xilinx mode, if a muxed IO's pad attribute is written through
```

Comment:
I think this differentiation between implementation targets is not ideal... in principle we should also add a check for the ASIC prims, but that `Impl<techname>` parameter is autogenerated, hence we cannot use it here in the code. This is more a note to myself. I will have to change that at some point.